lib/Conversion/TritonToLLVM/TritonToLLVM.cpp

```diff
@@ -225,6 +241,13 @@ void ConvertTritonToLLVMPass::runOnOperation() {
         typeConverter.convertType(type.getPointeeType()),
         type.getAddressSpace());
   });
+  typeConverter.addConversion([&](RankedTensorType type) -> Type {
```

Comment:
Add a comment that this should be only one out of several modes. Eventually, we want to handle pointers into one allocation as a base pointer with a tensor of offsets.